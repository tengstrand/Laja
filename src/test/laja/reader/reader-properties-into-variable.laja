
#write "target/test-output/reader-properties-into-variable.out"
  #read ("src/test/laja/reader/values.properties") into values
values.a = {values.a}
values.b = {values.b}
#end
