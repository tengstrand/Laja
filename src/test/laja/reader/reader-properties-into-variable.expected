values.a = abc
values.b = 123
